Library/Homebrew/service.rb

```diff
@@ -124,6 +124,16 @@ def command
       @run.map(&:to_s)
     end
 
+    sig { returns(String) }
+    def manual_command
+      instance_eval(&@service_block)
+      vars = @environment_variables.except(:PATH)
+                                   .map { |k, v| "#{k}=#{v}" }
```

Comment:
```suggestion
                                   .map { |k, v| "#{k}=\"#{v}\"" }
```

Is this a suggestion we should give? It may not be necessary, but I wonder if there are situations where the value would have whitespace.